src/test/java/gov/healthit/chpl/aqa/pageObjects/ScheduledJobsPage.java

```diff
@@ -0,0 +1,30 @@
+package gov.healthit.chpl.aqa.pageObjects;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Class ScheduledJobsPage definition.
+ */
+public final class ScheduledJobsPage extends BasePage {
+    private ScheduledJobsPage() {
+    }
+
+    public static WebElement typesOfSheduledJobs(final WebDriver driver) {
+        WebElement webelement  = driver.findElement(By.xpath("//table[@class='table table-condensed']//tbody"));
```

Comment:
I think there are two tables on this page that will meet this xpath, but you might not see the first one unless there are actually some jobs that are schedueld